subsys/mgmt/mcumgr/smp.c

```diff
@@ -143,36 +82,23 @@ static int
 zephyr_smp_tx_rsp(struct smp_streamer *ns, void *rsp, void *arg)
 {
 	struct zephyr_smp_transport *zst;
-	struct net_buf *frag;
 	struct net_buf *nb;
-	uint16_t mtu;
 	int rc;
-	int i;
 
 	zst = arg;
 	nb = rsp;
 
-	mtu = zst->zst_get_mtu(rsp);
-	if (mtu == 0U) {
+	if (zst->zst_get_mtu(rsp) == 0U) {
```

Comment:
This can be done by `zst_output` callback.

Reply:
Fixed